lib/index.js

```diff
@@ -0,0 +1,31 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import RequestController from '../controllers/RequestController';
+
+const app = express();
+
+const port = process.env.PORT || 8082;
+
+// parse application/json and look for raw text
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.text());
```

Comment:
Why are we using both bodyParser.text and bodyParser.json ?

Reply:
Just removed them now and found out, all my test still passed, I guess I was blindly following a tutorial. I only have this currently. Is that OK?
```
app.use(bodyParser.urlencoded({ extended: true }));
```

Reply:
👍